CONTRIBUTING.md

```diff
@@ -0,0 +1,21 @@
+# Second Screen Community Group
+
+This repository is being used for work in the W3C Second Screen Community Group, governed by the [W3C Community License Agreement (CLA)](https://www.w3.org/community/about/agreements/cla/). To make substantive contributions, you must join the CG. 
```

Comment:
join -> first join?

Reply:
This is a W3C template, so I wouldn't edit it.